### PR TITLE
VSCode Debugging launch template. e2e accept environment flags.

### PR DESCRIPTION
**Pull request by kaovilai**

### CODE REVIEW

File: docs/developer/testing/AttatchingIDEDebuggers.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    "configurations": [
+        {
+            "name": "Debug E2E Test",
+            "type": "go",
+            "request": "launch",
+            "mode": "test",
+            "program": "${workspaceRoot}/tests/e2e/e2e_suite_test.go",
+            "env": {
+                // "KUBECONFIG": "/path/to/.kube/config",
+                "KUBERNETES_MASTER": "http://localhost:8080",
```
Comment (weshayutin) [inline docs/developer/testing/AttatchingIDEDebuggers.md:22]: localhost:8080 4reelz?

Comment (kaovilai) [reply docs/developer/testing/AttatchingIDEDebuggers.md:22]: It's the default. Not defining it is also fine.

Comment (kaovilai) [reply docs/developer/testing/AttatchingIDEDebuggers.md:22]: Your kubeconfig takes care of what cluster it talks to.

**Comment by kaovilai** [comment]
Looks like openshift_ci need to be set to true for CI to create corresponding secrets required for e2e to work now.

**Pull request merged**